src/GTranslate/LanguageDictionary.cs

```diff
@@ -142,153 +142,167 @@ public bool TryGetLanguage(string code, [MaybeNullWhen(false)] out Language lang
+        ["kn"] = new("Kannada", "ಕನ್ನಡ", "kn", "kan"),
         ["ko"] = new("Korean", "한국어", "ko", "kor"),
         ["ku"] = new("Kurdish", "Kurdî", "ku", "kur", TranslationServices.Google | TranslationServices.Bing | TranslationServices.Microsoft),
         ["ky"] = new("Kyrgyz", "Kyrgyz", "ky", "kir"),
-        ["lo"] = new("Lao", "ລາວ", "lo", "lao"),
         ["la"] = new("Latin", "Latina", "la", "lat", TranslationServices.Google | TranslationServices.Yandex),
-        ["lv"] = new("Latvian", "Latviešu", "lv", "lav"),
-        ["lt"] = new("Lithuanian", "Lietuvių", "lt", "lit"),
         ["lb"] = new("Luxembourgish", "Lëtzebuergesch", "lb", "ltz", TranslationServices.Google | TranslationServices.Yandex),
-        ["mk"] = new("Macedonian", "Македонски", "mk", "mkd"),
+        ["ln"] = new("Lingala", "Lingála", "ln", "lin", TranslationServices.Google),
+        ["lo"] = new("Lao", "ລາວ", "lo", "lao"),
+        ["lt"] = new("Lithuanian", "Lietuvių", "lt", "lit"),
+        ["lv"] = new("Latvian", "Latviešu", "lv", "lav"),
+        ["lzh"] = new("Chinese (Literary)", "中文 (文言文)", "lzh", "lzh", TranslationServices.Bing | TranslationServices.Microsoft),
+        ["mai"] = new("Maithili", "Macushi", "mai", "mai", TranslationServices.Google),
         ["mg"] = new("Malagasy", "Malagasy", "mg", "mlg"),
-        ["ms"] = new("Malay", "Melayu", "ms", "msa"),
-        ["ml"] = new("Malayalam", "മലയാളം", "ml", "mal"),
-        ["mt"] = new("Maltese", "Malti", "mt", "mlt"),
+        ["mhr"] = new("Eastern Mari", "олык марий", "mhr", "mhr", TranslationServices.Yandex),
         ["mi"] = new("Maori", "Te Reo Māori", "mi", "mri"),
-        ["mr"] = new("Marathi", "मराठी", "mr", "mar"),
+        ["mk"] = new("Macedonian", "Македонски", "mk", "mkd"),
+        ["ml"] = new("Malayalam", "മലയാളം", "ml", "mal"),
         ["mn"] = new("Mongolian", "Монгол хэл", "mn", "mon"),
+        ["mn-Mong"] = new("Mongolian (Traditional)", "ᠮᠣᠩᠭᠣᠯ ᠬᠡᠯᠡ", "mn-Mong", "mn-Mong", TranslationServices.Bing | TranslationServices.Microsoft),
+        ["mr"] = new("Marathi", "मराठी", "mr", "mar"),
+        ["mrj"] = new("Western Mari", "Мары йӹлмӹ", "mrj", "mrj", TranslationServices.Yandex),
+        ["ms"] = new("Malay", "Melayu", "ms", "msa"),
+        ["mt"] = new("Maltese", "Malti", "mt", "mlt"),
+        ["mww"] = new("Hmong Daw", "Hmong Daw", "mww", "mww", TranslationServices.Bing | TranslationServices.Microsoft),
+        ["my"] = new("Burmese", "မြန်မာ", "my", "mya"),
         ["ne"] = new("Nepali", "नेपाली", "ne", "nep"),
+        ["nl"] = new("Dutch", "Nederlands", "nl", "nld"),
         ["no"] = new("Norwegian", "Norsk", "no", "nor"),
+        ["nso"] = new("Sepedi", "Sesotho sa Leboa", "nso", "nso", TranslationServices.Google),
+        ["ny"] = new("Chichewa", "Nyanja", "ny", "nya", TranslationServices.Google),
+        ["om"] = new("Oromo", "Afaan Oromoo [ʌfɑːn ɔrɔmoː] Oromiff", "om", "orm", TranslationServices.Google),
```

Comment:
The native language is just "Afaan Oromoo".
```suggestion
        ["om"] = new("Oromo", "Afaan Oromoo", "om", "orm", TranslationServices.Google),
```

Reply:
Fixed